packages/SwingSet/misc-tools/db-delete.js

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+// @ts-nocheck
```

Comment:
I was dismayed by all these `@ts-nocheck` until I realized that they weren’t previously checked and that you’ve just moved the exclusion from `jsconfig.json` to individual files, presumably so we can reveal them to the checker individually. Carry on.

Reply:
bingo. glad that was apparent enough since I was too lazy to leave a comment explaining